larpandoracontent/LArVertex/EnergyDepositionAsymmetryFeatureTool.cc

```diff
@@ -0,0 +1,193 @@
+/**
```

Comment:
A general comment about this class: As @MariaBrigida  pointed out to me, this class is extremely similar to the existing GlobalAsymmetryFeatureTool; the only difference appears to be in the CalculateEnergyDepositionAsymmetry function, corresponding to the CalculateGlobalAsymmetry function. I wonder if this might simply inherit from that class and simply override the CalculateGlobalAsymmetry function, or perhaps even modify the existing class to optionally compute the energy asymmetry instead?

Reply:
This is very true, there are quite strong overlaps with the other two asymmetry variables as well (shower and local). I could take a look at whether a base version of the tool with 4 inheriting versions might be neater?

Reply:
Sounds good thanks, Henry. Anything that can reasonably reduce code duplication without over-burdening a base class is useful for future maintenance.